Tests/RevenueCatUITests/CustomerCenter/Mocks/MockCustomerCenterPurchases.swift

```diff
@@ -13,7 +13,7 @@
 
 import Foundation
 import RevenueCat
-@testable import RevenueCatUI
+@_spi(Internal) @testable import RevenueCatUI
```

Comment:
These imports are now required in test files that used any `@_spi(Internal) public` API